### PR TITLE
fix: Updating logout function on README example code

### PR DESCRIPTION
**Pull request by maitzeth**
By submitting a PR to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](https://github.com/auth0/.github/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

Updating README documentation example code. The `returnTo` parameter has been moved into `logoutParams`.

### References

[#550 ](https://github.com/auth0/auth0-react/issues/550)

### Testing

N/A

- [ ] This change adds test coverage for new/changed/fixed functionality

### Checklist

- [x] I have added documentation for new/changed functionality in this PR or in auth0.com/docs
- [ ] All active GitHub checks for tests, formatting, and security are passing
- [ ] The correct base branch is being used, if not the default branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,7 +128,13 @@ function App() {
...
       <div>
         Hello {user.name}{' '}
-        <button onClick={() => logout({ returnTo: window.location.origin })}>
+        <button onClick={() => {
+          return logout({
+            logoutParams: {
+              returnTo: window.location.origin,
+            },
+          });
+        }}>
```
Comment (frederikprijck) [inline README.md:137]: Just a little nit to better align with our docs elsewhere
```suggestion
        <button onClick={() => logout({ logoutParams: { returnTo: window.location.origin } })}>
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

**Review by frederikprijck — APPROVED** [review]
Thanks! 🚀

File: README.md (hunk 1 above)
Comment (maitzeth) [reply README.md]: ty, till' next time :)

**Comment by maitzeth** [comment]
@frederikprijck looks like something went wrong with CI :)

**Pull request merged**